chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -947,3 +947,93 @@ Example
 - Check WMS with Swiss ech-0056 profile (html): `https://api3.geo.admin.ch/owschecker/form?base_url=http%3A%2F%2Fwms.zh.ch%2Fupwms&service=WMS <../../../owschecker/form?base_url=http%3A%2F%2Fwms.zh.ch%2Fupwms&service=WMS>`_
 - Check WMTS with Swiss ech-0056 profile (xml): `https://api3.geo.admin.ch/owschecker/bykvp?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS <../../../owschecker/bykvp?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS>`_
 - Check WMTS with Swiss ech-0056 profile (html): `https://api3.geo.admin.ch/owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS <../../../owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS>`_
+
+
+
+
+.. _owschecker_description:
+
+----------
+
+Stationboard
+------------
+
+This service provides the next connections from a location to a specific destination or all destinations for a given location. 
+This service uses public transportation timetable data and not real time information.
+
+URL
+***
+
+::
+
+  https://api3.geo.admin.ch/stationboard/stops/{id} OR
+  https://api3.geo.admin.ch/stationboard/stops/{id}/destinations
+
+Input Parameters
+****************
+
++-----------------------------------+-------------------------------------------------------------------------------------------+
+| Parameters                        | Description                                                                               |
++===================================+===========================================================================================+
+| **destination (required)**        | Specifies the arrival location of the connection                                          |
```

Comment:
mmhh, I just noticed now, why do we use the id once and a string the next time?
`destination` parameter should be an id as well.... Can you adapt? You realize you're asking for an exact match string match?